lib/rails_response_dumper/runner.rb

```diff
@@ -20,7 +20,12 @@ def run_dumps
           dumper.mock_setup
           begin
             rollback_after do
-              dumper.instance_eval(&dump_block.block)
+              dumper.instance_eval(&defined.before_block) if defined.before_block
+              begin
+                dumper.instance_eval(&dump_block.block)
+              ensure
```

Comment:
Could a test be added that demonstrates that when the dump raises an exception, the after hook still runs?

Reply:
The runner does not continue after an exception is raised, so I wasn't able to figure a way to test this. Open to suggestions anyone has

Reply:
Could we have the after block write a file that is then asserted on? Similar to https://github.com/Pioneer-Valley-Books/rails-response-dumper/blob/main/spec/cli_spec.rb#L10

Reply:
@jrdufresne, the exception would still prevent the rest of the dumps from being run though, right? Let me know if I'm missing something

Reply:
Since it is in the `ensure` it should always run regardless of the dump result.